prep-yml-bundle/replace-vars.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/ruby
+
+# Usage ./replace_vars.rb COMMA,SEPARATED,VAR_NAMES path/to.yml [path/to.yml ... ]
+
+# This script will walk through all yaml files provided, and for any object that has a key 'name' that matches
+# the name of a variable to replace, it will replace the value of 'value' in the yaml with the value of the
+# variable from the environment. Requires Ruby >= 2.1.0
```

Comment:
Line wrap problems (a bit too wide).

Reading the code, this actually only changes `spec/template/spec/containers/env/*`, not what is described at all.  The description made me think it would do so regardless of where in the hierarchy the key is.